src/main/java/org/opensearch/ad/client/AnomalyDetectionClient.java

```diff
@@ -110,4 +114,40 @@ default ActionFuture<SuggestConfigParamResponse> suggestAnomalyDetector(SuggestC
      * @param listener a listener to be notified of the result
      */
     void suggestAnomalyDetector(SuggestConfigParamRequest suggestRequest, ActionListener<SuggestConfigParamResponse> listener);
+
+    /**
+     * Create anomaly detector - refer to https://opensearch.org/docs/latest/observing-your-data/ad/api/#create-detector
+     * @param createRequest request to create the detector
+     * @return ActionFuture of IndexAnomalyDetectorResponse
+     */
+    default ActionFuture<IndexAnomalyDetectorResponse> createAnomalyDetector(IndexAnomalyDetectorRequest createRequest) {
+        PlainActionFuture<IndexAnomalyDetectorResponse> actionFuture = PlainActionFuture.newFuture();
+        createAnomalyDetector(createRequest, actionFuture);
+        return actionFuture;
+    }
+
+    /**
+     * Create anomaly detector - refer to https://opensearch.org/docs/latest/observing-your-data/ad/api/#create-detector
+     * @param createRequest request to create the detector
+     * @param listener a listener to be notified of the result
+     */
+    void createAnomalyDetector(IndexAnomalyDetectorRequest createRequest, ActionListener<IndexAnomalyDetectorResponse> listener);
+
+    /**
+     * Start anomaly detector - refer to https://opensearch.org/docs/latest/observing-your-data/ad/api/#start-detector
```

Comment:
Should be https://opensearch.org/docs/latest/observing-your-data/ad/api/#start-detector-job